### PR TITLE
Fix: Angry Mob spawns rocks and molotovs from feet, rocks disappear after impact

### PR DESCRIPTION
**Pull request by commy2**
### before

https://github.com/TheSuperHackers/GeneralsGamePatch/assets/6576312/2da17fcb-c232-48fb-9916-690f1734e5d2

### after

https://github.com/TheSuperHackers/GeneralsGamePatch/assets/6576312/3a5635ac-27ad-4729-867a-2e20c7306f25

### CODE REVIEW

**Comment by commy2** [comment]
Bouncing rocks.

https://github.com/TheSuperHackers/GeneralsGamePatch/assets/6576312/23331c42-dc6b-43c5-927a-5095dcc1adbe



https://github.com/TheSuperHackers/GeneralsGamePatch/assets/6576312/2b40e9ca-7f29-40ba-baa5-67fc79ef36a2

File: Patch104pZH/GameFilesEdited/Data/INI/Object/GLAInfantry.ini [hunk 1]
```diff
@@ -2184,6 +2184,7 @@ Object GLAInfantryAngryMobRock02
       AnimationSpeedFactorRange = 2.5 2.5
       TransitionKey = TRANS_FIRING_A
       WaitForStateToFinishIfPossible = NoWait
+      WeaponLaunchBone = PRIMARY "UIMOB02 R HAND" ; Patch104p @bugfix commy2 23/08/2023 Fix projectile spawning from feet. (#xxxx)
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/GLAInfantry.ini]: xxxx

**Review by xezon — APPROVED** [review]
Very nice. Needs documentation.

**Comment by MTKing4** [comment]
Why is your game lagging so hard?

**Comment by commy2** [comment]
Laptop low battery.

**Comment by MTKing4** [comment]
Ah ok

**Pull request merged**